server/xpub-model/entities/file/index.test.js

```diff
@@ -35,19 +35,53 @@ describe('Manuscript', () => {
       const response = await file.delete()
       expect(response).toBeTruthy()
     })
+  })
 
+  describe('updateStatus()', () => {
     it('should be initialised in the CREATED state', async () => {
-      const manuscript = new Manuscript({
+      const manuscript = await new Manuscript({
         createdBy: userId,
-      })
-      await manuscript.save()
-      const file = new File({
+      }).save()
+      const file = await new File({
         manuscriptId: manuscript.id,
         filename: 'thisfile.txt',
         url: '/an/url',
-      })
-      await file.save()
+      }).save()
+      expect(file.status).toBe('CREATED')
+    })
+    it('sets file status to value passed', async () => {
+      const manuscript = await new Manuscript({
+        createdBy: userId,
+      }).save()
+      let file = await new File({
+        manuscriptId: manuscript.id,
+        filename: 'thisfile.txt',
+        url: '/an/url',
+      }).save()
+      expect(file.status).toBe('CREATED')
+      file = await File.find(file.id)
+      await file.updateStatus('UPLOADED', userId)
+      expect(file.status).toBe('UPLOADED')
+      file = await File.find(file.id)
+      await file.updateStatus('CANCELLED', userId)
+      expect(file.status).toBe('CANCELLED')
+    })
+  })
+
+  describe('save()', () => {
```

Comment:
This whole `describe` can be removed as the `File.save()` no longer calls a `manuscript.validate()`. This is my bad, should have actually failed in the original PR too